modules/test/playwright/utils/performLogin.ts

```diff
@@ -46,12 +46,18 @@ async function performLogin(
 
 	await page.goto(baseUrl);
 
-	const signInButton = await page.getByRole('button', {name: 'Sign In'});
+	const signInButton = page.getByRole('button', {name: 'Sign In'});
+
+	await expect(page.getByPlaceholder('Search')).toBeVisible();
```

Comment:
This fix flakiness in login. Flakiness is caused by the fact that `Sign In` button is not usable for a short time when the page is first loaded. We are now waiting for `Search` input to be visible, using it as an indicator that hydration completed.

Reply:
:+1: The previous code was copied from the very first PoC so I assumed it was correct. If it was flaky and this change makes waiting more reliable it's great :+1: